docker-compose.override.yml

```diff
@@ -2,9 +2,14 @@ version: "3.5"
 services:
   temporal:
     environment:
-      - PROMETHEUS_ENDPOINT=0.0.0.0:9090
+      - PROMETHEUS_ENDPOINT=0.0.0.0:8000
+    ports:
+      - 8000:8000
```

Comment:
You don't need to expose the port, only prometheus is going to hit it and prometheus talks to the container directly.